pkg/apis/stream/v1alpha1/processor_types.go

```diff
@@ -42,10 +42,11 @@ type ProcessorSpec struct {
 type ProcessorStatus struct {
 	duckv1alpha1.Status `json:",inline"`
 
-	InputAddresses  []string `json:"inputAddresses,omitempty"`
-	OutputAddresses []string `json:"outputAddresses,omitempty"`
-	DeploymentName  string   `json:"deploymentName,omitempty"`
-	FunctionImage   string   `json:"functionImage,omitempty"`
+	InputAddresses     []string `json:"inputAddresses,omitempty"`
+	OutputAddresses    []string `json:"outputAddresses,omitempty"`
+	OutputContentTypes []string `json:"outputAddresses,omitempty"`
```

Comment:
Copy/paste is the root of all evil :) see json tag

Reply:
oopsie :)